src/main/java/net/coderbot/iris/config/IrisConfig.java

```diff
@@ -107,6 +108,7 @@ public void load() throws IOException {
 		properties.load(Files.newInputStream(propertiesPath));
 		shaderPackName = properties.getProperty("shaderPack");
 		enableShaders = !"false".equals(properties.getProperty("enableShaders"));
+		IrisVideoSettings.shadowDistance = Integer.parseInt(properties.getProperty("shadowRenderDistance", "0"));
```

Comment:
We probably want to add error checking here so that an invalid value gets reset instead of crashing. Also, a small name change. Finally, this should in fact default to 32 - allowing people to effectively disable shadows by setting the distance to zero is desirable.
```suggestion
		IrisVideoSettings.shadowDistance = Integer.parseInt(properties.getProperty("maxShadowRenderDistance", "32"));
```